frappe/core/doctype/system_settings/system_settings.json

```diff
@@ -485,12 +487,25 @@
    "fieldtype": "Select",
    "label": "First Day of the Week",
    "options": "Sunday\nMonday\nTuesday\nWednesday\nThursday\nFriday\nSaturday"
+  },
+  {
+   "default": "90",
+   "description": "Number of days after which the shared document link will be expired",
+   "fieldname": "document_share_key_expiry",
+   "fieldtype": "Int",
+   "label": "Document Share Key Expiry (in Days)"
+  },
+  {
+   "default": "1",
```

Comment:
Let's keep this 0 and make a patch. This way, the insecure setting doesn't get enabled for new users.